package.json

```diff
@@ -13,21 +13,21 @@
   "license": "Talk to the Legal Department",
   "main": "NestHydrationJS.js",
   "scripts": {
-    "coverage": "node_modules/istanbul/lib/cli.js cover node_modules/jasmine/bin/jasmine.js",
```

Comment:
On my local this change results in the following error:

```
No coverage information was collected, exit without writing coverage information
module.js:442
    throw err;
    ^

Error: Cannot find module 'node_modules/.bin/jasmine'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Function.Module.runMain (module.js:575:10)
    at runFn (/Users/me/work/nesthydrationjs/node_modules/istanbul/lib/command/common/run-with-cover.js:122:16)
    at /Users/me/work/nesthydrationjs/node_modules/istanbul/lib/command/common/run-with-cover.js:251:17
    at /Users/me/work/nesthydrationjs/node_modules/istanbul/lib/util/file-matcher.js:68:16
    at /Users/me/work/nesthydrationjs/node_modules/async/lib/async.js:52:16
    at /Users/me/work/nesthydrationjs/node_modules/async/lib/async.js:361:13
    at /Users/me/work/nesthydrationjs/node_modules/async/lib/async.js:52:16
    at done (/Users/me/work/nesthydrationjs/node_modules/async/lib/async.js:246:17)
```

I think it might be this issue https://github.com/jasmine/jasmine-npm/issues/60 but it isn't fixed yet. Please switch back this regression.